docs/guides/getting-started-java-spring.md

```diff
@@ -6,7 +6,7 @@ description: "Use Spring Boot and the Spring Zeebe SDK to interact with your loc
 keywords: [java, spring, spring zeebe, getting started, user guide, tutorial]
 ---
 
-import Install from './react-components/install-c8run.md'
+import Install from './react-components/\_install-c8run.md'
```

Comment:
I updated the branch to resolve the build issues, but is the `\` required? I was just expecting `_`. Can you elaborate on this if it's not a mistake?

Reply:
It isn't, and when I thought it was only this line, I assumed I'd just been typing too fast - but I am not sure if these were all added by pre-commit linting?

Reply:
@akeller So prettier(?) is inserting the backslash to acknowledge that this is an intentional/literal underscore, and not meant to be parsed as italics. When I test locally with the backslash, it seems to still be able to resolve the link and insert the partial, but I don't know if that's just local or a quirk! I added the deploy label to check the behavior.

@pepopowitz Do you happen to know if leaving this backslash will operate any differently when live, or if there's a better method here than letting docusaurus + prettier figure it out?

Reply:
1. It is definitely prettier that wants the underscore to be escaped.
2. It doesn't appear to be a problem. [Here is a similar change in the PR](https://github.com/camunda/camunda-docs/pull/4448/files#diff-acfeef816ca131e1db6b1bd1e1d8d6db10d86b45910f821fa3597c561e26d628L15), and I can see on [the rendered page](https://preview.docs.camunda.cloud/pr-4448/docs/guides/orchestrate-apis/) that the partial is displayed. 
3. I think something got mixed up on this specific example. That partial is imported...but I don't see it emitted anywhere on the page? I'd expect there to be a `<Install />` on the page, but I can't find it. I also don't see explicit instructions for installing C8 run [on the rendered page](https://preview.docs.camunda.cloud/pr-4448/docs/guides/getting-started-java-spring/). It also seems like _some_ of the contents of the partial are duplicated inline on the page. Is this expected?

Reply:
More specifically, re "if there's a better method here than letting docusaurus + prettier figure it out", what we're doing in this PR seems correct, and seems to work, so I wouldn't change anything.

Reply:
> I think something got mixed up on this specific example. 
This pit in my stomach 🤮 .... there are a lot of cooks in the kitchen with this guide, but we are ok _not_ having the partial because we link to /self-managed/setup/deploy/local/c8run.md in the pre-requisites (line 40). 

Ultimately, I think @conceptualshark wants to have these install steps in a partial and included in the guide, but UX feedback from @camundom (via https://github.com/camunda/camunda-docs/pull/4301) was to remove them. A good topic to follow up on because I strongly believe they should be included, particularly as a partial. @conceptualshark please follow up on this.

Reply:
^ Agreed, I think this is (at the moment) left over from the guide being restructured, and the partial isn't actually used at all right now. I can follow up with cleaning this up/seeing if we want the install steps there in full or not (perhaps based on additional user feedback?).